tmt/cli.py

```diff
@@ -129,7 +129,7 @@ def implemented_tested_documented(function):
 @click.pass_context
 @click.option(
     '-r', '--root', metavar='PATH', show_default=True,
-    help='Path to the tree root. \'.\' by default.')
+    help="Path to the tree root, '.' by default.")
```

Comment:
Even though using double quotes for the string is more readable, all help strings use single quotes. Wouldn't it be better to stay consistent at the cost of slight decrease in readability?

Reply:
See your point. But I'd say say here the readability is a little bit more important than keeping all quotes the same.